mvvmfx/src/main/java/de/saxsys/mvvmfx/internal/viewloader/FxmlViewLoader.java

```diff
@@ -15,10 +15,7 @@
  ******************************************************************************/
 package de.saxsys.mvvmfx.internal.viewloader;
 
-import de.saxsys.mvvmfx.Context;
-import de.saxsys.mvvmfx.Scope;
-import de.saxsys.mvvmfx.ViewModel;
-import de.saxsys.mvvmfx.ViewTuple;
+import de.saxsys.mvvmfx.*;
```

Comment:
please undo this format action. I personally don't like *-imports because they hide what is actually imported. This makes reviewing code outside of IDEs harder.